### PR TITLE
Add - Home Page

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Add Quiz Home Page

Preview - https://deploy-preview-1--vishnu-quiz.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            href="/pages/rules.html"
+            >Play Now</a
+          >
+        </div>
+        <div class="card__body p-2">
+          <div class="card__img-wrapper">
+            <img
+              class="card__img"
+              src="/assets/Images/suits.jpg"
+              alt="Card Header Image"
```
Comment (partha8) [inline index.html:83]: You can avoid writing "image" in the alt attribute as its already known that its an image right?

**Review by partha8 — APPROVED** [review]
Looks good. The indentation of the code can be improved for better readability.

File: index.html (hunk 1 above)
Comment (Vishnu-Aithal) [reply index.html:83]: oh ok k. will remove it!
Thanks for Reviewing

**Comment by Vishnu-Aithal** [comment]
> Looks good. The indentation of the code can be improved for better readability.

I have used Prettier default settings. will reconfigure it

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

**Pull request merged**